identity/src/main/java/com/android/identity/android/util/AndroidLogPrinter.kt

```diff
@@ -0,0 +1,20 @@
+package com.android.identity.android.util
+
+import android.util.Log
+import com.android.identity.util.Logger.LEVEL_D
+import com.android.identity.util.Logger.LEVEL_E
+import com.android.identity.util.Logger.LEVEL_I
+import com.android.identity.util.Logger.LEVEL_W
+import com.android.identity.util.Logger.LogPrinter
+
+/** An implementation of [LogPrinter] that logs to logcat. */
```

Comment:
is `[LogPrinter]` something Kotlin specific? The Java way to do this is `{@link LogPrinter}`

Reply:
Yeah, kotlin uses '[...]' rather than {@link ...} Now that we are using dokka this also works, yay! https://kotlinlang.org/docs/kotlin-doc.html#links-to-elements